infrastructure/k8s/helm-chart/eoapi/templates/db/manifest.yaml

```diff
@@ -1,9 +1,12 @@
+{{- include "eoapi.validateValues" . }}
+---
 apiVersion: v1
-kind: ConfigMap
+kind: Secret
 metadata:
-  name: pgstac-env-configmap
-data:
-  {{- range $envKey, $envValue := .Values.db.settings.envVars }}
+  name: pgstac-secrets
+type: "Opaque"
+stringData:
+  {{- range $envKey, $envValue := .Values.db.settings.secrets }}
```

Comment:
comment for reviewer: we now flow through os env variables to a `kind: Secret` since it's a standard. All deployments and services now can either mount these or look them up and bring them in as their own os env variables